### ISSUE TITLE
loss is nan from very first epoch while running Blood Cell Detection.ipynb

### ISSUE THREAD

**Issue by shirishr** [experiencor/keras-yolo2#222]
@experiencor 
Thank you for fixing #221. Now I run into another one.

Is some correction needed in the custom loss?
I am seeing:

Epoch 1/100
69/69 [==============================] - 43s 619ms/step - loss: nan - val_loss: nan
Epoch 00001: val_loss did not improve

Eventually val_loss never improves and early stopping after 3rd epoch makes the model un-trained

Thanks in advance !!

**Comment by experiencor** [comment]
@shirishr Are you using the notebook. It does not include warmup training. Recommend using the python code.

**Comment by shirishr** [comment]
@experiencor,

Thank you sir

Yes indeed I am using the notebook !
It does have some small code called

    """
    Warm-up training
    """
Which python code are you suggesting?
I will try to follow steps from **Yolo Step-by-Step.ipynb**  instead (they are slightly different from the steps in **examples/Blood Cell Detection.ipynb** where I get errors)

I will also try and follow steps in README.md but use the Blood Cell data. Is that what you mean when you say follow the python code?

Thanks

**Comment by experiencor** [comment]
@shirishr Yes, I mean following the instruction in the README. The notebook is outdated as it does not include the warmup stage.

**Comment by tamersalama** [comment]
Trying to update the nan related issues - what worked for me is setting having images and annotations added to "valid_image_folder" (I relied on having the training ones split 80/20 as per the readme - but I got nan for losses). I also changed the traing nb_epochs to 10 - and will likely need more (from 1)
